index.html

```diff
@@ -1,16 +1,23 @@
-<html>
+﻿<html>
 <head>
-  <meta charset="UTF-8">
-  <script language="javascript" type="text/javascript" src="libraries/p5.js"></script>
-  <script language="javascript" type="text/javascript" src="libraries/p5.dom.js"></script>
-  <script language="javascript" type="text/javascript" src="libraries/p5.sound.js"></script>
-  <script language="javascript" type="text/javascript" src="sketch.js"></script>
-  <script language="javascript" type="text/javascript" src="input.js"></script>
-  <script language="javascript" type="text/javascript" src="entity.js"></script>
-  <script language="javascript" type="text/javascript" src="ship.js"></script>
-  <script language="javascript" type="text/javascript" src="asteroid.js"></script>
-  <script language="javascript" type="text/javascript" src="laser.js"></script>
-  <style> body {padding: 0; margin: 0;} </style>
+    <title>Community Asteroids! - Coding Rainbow</title>
+    <meta charset="UTF-8">
+    <script language="javascript" type="text/javascript" src="libraries/p5.js"></script>
+    <script language="javascript" type="text/javascript" src="libraries/p5.dom.js"></script>
+    <script language="javascript" type="text/javascript" src="libraries/p5.sound.js"></script>
+    <script language="javascript" type="text/javascript" src="sketch.js"></script>
+    <script language="javascript" type="text/javascript" src="input.js"></script>
+    <script language="javascript" type="text/javascript" src="entity.js"></script>
+    <script language="javascript" type="text/javascript" src="ship.js"></script>
+    <script language="javascript" type="text/javascript" src="hud.js"></script>
+    <script language="javascript" type="text/javascript" src="asteroid.js"></script>
+    <script language="javascript" type="text/javascript" src="laser.js"></script>
+    <style>
+        body {
+            padding: 0;
+            margin: 0;
+        }
+    </style>
 </head>
```

Comment:
Maybe not change the indentation on everything so it doesn't go all green and red on us.